druid-shell/src/mac/mod.rs

```diff
@@ -53,6 +57,7 @@ pub struct WindowHandle {
     /// TODO: remove option (issue has been filed against objc, or we could manually impl default with nil)
     /// https://github.com/SSheldon/rust-objc/issues/77
     nsview: Option<WeakPtr>,
+    queue: Weak<Mutex<Vec<Box<IdleCallback>>>>,
```

Comment:
`queue` is a bit of a weak name, but `idle_callback_queue` might be too long...

Reply:
Actually I might want to reconsider "idle", as that suggests a lower priority than other events on the UI thread, and I don't think that's accurate. The name is taken from gtk. But I think "idle_queue" is probably the best compromise for now.